### DISCUSSION TITLE
Geojson with elevation coordinate not drawn by harp.gl.

### DISCUSSION THREAD

**Discussion by utkarsh2794** [heremaps/harp.gl#2111] [Q&A] [answered]
don't know if there is a way to draw 3d buildings with jeojson multipolygon with z axis. with harp gl was only able to manage 2d building with height data present.
also is there a way i can represent the floor level with labels in each floor from properties data in Geojson.

**Comment by FraukeF** [comment, marked as answer]
Hi @utkarsh2794 

Unfortunately, we do not support neither of these features at this point. 
GeoJson with elevation data, in works for already tiled geojson, if the geojson provided needs tiling still, at the current point the tiling library we are using, will remove the altitude. Still, I guess it wont work with the extrusion.
